CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file based on the
 #### Bugfixes
 
 * The `protocol` allowed value under `event.type` should not have the `expected_event_types` defined. #964
+* Clarify the definition of `file.extension` (no dots). #1016
```

Comment:
@ebeahan Since we'll backport this all the way back to 1.6/current, I'm shoving this PR's changelog directly in the 1.7 changelog for now.

I'll add a separate entry in 1.6 on the backport.

WDYT of this approach?

A bug fix changing the structure of the events (type change, typo fix) that we'd backport would warrant a point release. But clarifying the description to make explicit what was hidden in the example doesn't warrant a point release, IMO.

However I think it's important to make the change visible to anyone looking at past changelogs.

Reply:
I agree with this approach of modifying `CHANGELOG.md` directly and then adding a separate entry for the `CHANGELOG.md` in the 1.6 branch.

Also agree that since the change only improves the description around what value was already expected, this doesn't warrant the point release.